tests/credentialDefinition.test.ts

```diff
@@ -82,12 +85,55 @@ describe('CredentialDefinitionController', () => {
     })
 
     test('should return 404 NotFound when credential definition not found', async () => {
+      const spy = stub(agent.modules.anoncreds, 'getCredentialDefinition')
+      spy.resolves({
+        credentialDefinitionId: 'x',
+        credentialDefinitionMetadata: {},
+        resolutionMetadata: {
+          error: 'notFound',
+        },
+      })
+
```

Comment:
how was the test passing before, without the stub?

Reply:
It wasn't? - I just ran tests on main and it failed there too.

Reply:
weird, idk how that got into `main`. Good that this fixes it anyway